WalletWasabi/KeyManagement/KeyManager.cs

```diff
@@ -69,10 +75,11 @@ public class KeyManager
 		public bool IsHardwareWallet => EncryptedSecret is null && MasterFingerprint != null;
 		public HardwareWalletInfo HardwareWalletInfo { get; set; }
 
-		private const int AbsoluteMinGapLimit = 21;
+		public const int AbsoluteMinGapLimit = 21;
+		public const int AbsoluteMaxGapLimit = 1_000_000;
```

Comment:
Uhmm... if someone sets the `minGapLimit` to a 1e6 it could take a long time to derivare that number of keys. Have you measured how much it takes?

Reply:
No, I didn't try it myself. @nopara73 suggested to add the upper bound of 1 million. It seemed large to me too but who am I to disagreee ;-)